packages/snowflake/src/lib/Snowflake.ts

```diff
@@ -1,7 +1,24 @@
 /* eslint-disable @typescript-eslint/explicit-member-accessibility */
 
+import { threadId } from 'node:worker_threads';
+
+// For testing purposes, we'll assign processId and workerId
+// to 1n so it's easier to match the generated snowflakes:
+const ProcessId = process.env.NODE_ENV === 'test' ? 1n : BigInt(process.pid);
+const WorkerId = process.env.NODE_ENV === 'test' ? 0n : BigInt(threadId);
```

Comment:
This should be done through Jest spies/mocks and not incur changes on production code.

```suggestion
// For testing purposes, we'll assign processId and workerId
// to 1n so it's easier to match the generated snowflakes:
const ProcessId = BigInt(process.pid);
const WorkerId = BigInt(threadId);
```

And then overwrite them with spies/mocks

Reply:
You tell me how I overwrite them.